docs/content/r&d/services/boot/admin-api.md

```diff
@@ -0,0 +1,796 @@
+---
+title: "HTTP REST Admin API"
+type: docs
+description: "Management API for boot images, machines, and profiles"
+weight: 20
+---
+
+The Admin API provides HTTP REST endpoints for managing boot images, machine mappings, and boot profiles. All endpoints require GCP IAM authentication.
+
+## Authentication
+
+All admin API endpoints require authentication using one of the following methods:
+
+- **Service Account Token**: `Authorization: Bearer <service-account-token>`
+- **User OAuth Token**: `Authorization: Bearer <user-oauth-token>`
+- **IAM Authentication**: Validated against GCP IAM policies
+
+**Required IAM Permissions:**
+
+- `bootserver.images.create` - Upload boot images
+- `bootserver.images.read` - List and retrieve boot images
+- `bootserver.images.delete` - Delete boot images
+- `bootserver.machines.create` - Register machines
+- `bootserver.machines.read` - List and retrieve machine configurations
+- `bootserver.machines.update` - Update machine mappings
+- `bootserver.machines.delete` - Delete machine registrations
+- `bootserver.profiles.create` - Create boot profiles
+- `bootserver.profiles.read` - List and retrieve profiles
+- `bootserver.profiles.update` - Update boot profiles
+- `bootserver.profiles.delete` - Delete boot profiles
+
+---
+
+## Boot Image Management
+
+### `POST /api/v1/images`
```

Comment:
the actual boot image assets should be uploaded to this API, not links to remote storage locations. This API is the source of truth for storage and retrieving boot assets